modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/TrafficSources.js

```diff
@@ -173,8 +173,10 @@ export default function TrafficSources({dataProvider, onTrafficSourceClick}) {
 						<tbody>
 							{trafficSources?.map((entry) => {
 								const hasDetails =
-									entry?.value > 0 && entry.name !== 'direct';
-
+									entry?.value > 0 &&
+									(entry.name === 'referral' ||
```

Comment:
Hey @Stephy6 and @claraizquierdo , what do you think about using a more generic approach, like if the endpoint is available or not?